regexes/bots.yml

```diff
@@ -2118,6 +2124,67 @@
   category: 'Crawler'
   url: 'http://www.exensa.com/crawl'
 
+- regex: 'BDCbot'
+  name: 'BDCbot'
+  category: 'Crawler'
+  url: 'https://bigweb.bigdatacorp.com.br/pages/faq.aspx'
+  producer:
+    name: 'BIG Data Solucoes Em Tecnologia de Informatica LTDA'
+    url: 'https://bigdatacorp.com.br/'
+
+- regex: 'adbeat'
+  name: 'Adbeat'
+  category: 'Crawler'
+  url: 'https://www.adbeat.com/operation_policy'
+  producer:
+    name: 'PPC Labs LLC'
+    url: 'https://www.adbeat.com/'
+
+- regex: 'BW'
```

Comment:
2 characters is very small, you need to add one more condition to avoid collisions in the future.
e.g.
```yml
BM/(?:(\d+[\.\d]+))
```